Script/JAX/tooltip.ts

```diff
@@ -38,6 +39,10 @@ export default function toolTip(el) {
         document.querySelector('#page').appendChild(tooltip);
     }
 
+    if (el.hasAttribute('data-last-online')) {
+        title += emojiTime(el.getAttribute('data-last-online'));
```

Comment:
while you're in here, please change the whole title to the user's relative time?

something like this would probably do it:
```javascript

const timestamp= el.getAttribute('data-last-online')
title = emojiTime(timestamp) + date(timestamp);
```

I just realized the titles are coming from the server and aren't translated to the user's local time